data/data-pipeline/data_pipeline/etl/score/etl_score.py

```diff
@@ -299,6 +326,13 @@ def _prepare_initial_df(self) -> pd.DataFrame:
             field_names.EXPECTED_BUILDING_LOSS_RATE_FIELD_NAME,
             field_names.EXPECTED_AGRICULTURE_LOSS_RATE_FIELD_NAME,
             field_names.EXPECTED_POPULATION_LOSS_RATE_FIELD_NAME,
+            field_names.CENSUS_DECENNIAL_HIGH_SCHOOL_ED_FIELD_2009,
```

Comment:
This isn't blocking, but I'm not stoked about the inconsistent naming here -- we've got some that end with `FIELD_NAME`, others that end with `FIELD_2009`, and then others that just end with the year (e.g. `_2009`). Is there a rationale behind these differences?

Reply:
We can update them. I wanted to make it super clear for developers which data is "old" (from 2009), it can be a little confusing between the high school education field from 2019 versus the high school education field from 2009. But perhaps there's a better way to do this. 

Do you have any suggestions for naming conventions that are more consistent but still flag to future developers, this data is old? The `CENSUS_DECENNIAL` helps a little.

Reply:
Makes sense. I like having the year last to make it pop out more -- or even putting it first. But for consistency in using `FIELD` or `FIELD_NAME`, maybe they should all just use `FIELD`? That would mean updating lines 331, 332, and 334 to have it, and 326, 327, and 328 to remove `_NAME`.

Reply:
yeaaaa having both FIELD and FIELD_NAME is dumb. I'll update! Good flag!!